docs/vgclog_examples.md

```diff
@@ -0,0 +1,594 @@
+|----------|----------------------------|------------------------------|------------------|
+|Global    | `global`                   | `<concurrent-kickoff>`       | Memory threshold reached. Cycle trigger element is located before the `<cycle-start>` element|
+|Partial | `scavenge`                   | `<af-start>`                 |Allocation failure. Cycle trigger element is located before the `<cycle-start>` element|
+
+The element that records the cycle trigger is located before the '<cycle-start>` element and can be used to locate a particuler type of cycle. 
+
+You can analyze the increments and operations that are associated with a particular type of cycle by locating and interpreting the elements in the following table:
+
+
+| GC process             | Elements that log the start and end of the event  | Details |
+|------------------------|------------------------------|------------------------------|
+|GC cycle                |`<cycle-start>`, `<cycle-end>`| The start and end of a GC cycle.|
+|GC STW increment        |`<gc-start>`, `<gc-end>`      | The start and end of a GC increment that begins with a pause.
+|GC STW increment        | `<concurrent-kickoff>`       | The start of the initial GC increment of the global concurrent cycle that begins the initial mark operation
+|GC STW increment        | `<concurrent-global-final> ` | The start of the final GC increment of the global concurrent cycle that executes the final collection|
+|GC operations and sub-operations| `<gc-op>`                    | A GC operation such as mark or sweep, or a suboperation such as class unload. |
+
+**Note:** For more information about the XML structure of GC cycles, see [GC cycles](vgclog.md/#gc-cycles). For more information about GC cycle increments, see [GC increments and interleaving](vgclog.md#gc-increments-and-interleaving).
+
+The following examples use log excerpts to show how the different types of `gencon` cycle are logged.
+
+### Scavenge partial GC cycle
+
+The following example is taken from a `gencon` log. The output is broken down into sections with supporting text to explain the GC processing that is taking place.
+
+By default, the `gencon` partial GC cycle runs by using a single *stop-the-world* (STW) pause. The cycle performs only 1 operation,  a *scavenge* operation, which runs only on the *nursery* area. The cycle consists of a single GC increment, which is labelled using the elements shown in the following table:
+
+|GC Operation | GC increment | STW or concurrent| XML element of GC increment          | Details                                                                   |
+|-------------|--------------|------------------|--------------------------------------|---------------------------------------------------------------------------|
+|Scavenge     |Single        | STW              | `<gc-start>`, `<gc-end>` |Contains detailed information about root scanning and weak roots processing|
```

Comment:
We don't provide detailed information about (strong) root scanning, but about objects being copied and about weak roots processing.